app/javascript/packages/verify-flow/steps/personal-key-confirm/personal-key-confirm-step.spec.tsx

```diff
@@ -58,6 +58,22 @@ describe('PersonalKeyConfirmStep', () => {
     expect(analytics.trackEvent).to.have.been.calledWith('IdV: hide personal key modal');
   });
 
+  it('passes the value the user has entered to this point to the child PersonalKeyInput', () => {
+    const props = Object.assign(DEFAULT_PROPS, {
+      value: {
+        personalKey: '',
+        personalKeyConfirm: '1234-asdf',
+      },
+      toPreviousStep: () => {},
+    });
```

Comment:
I would have thought the linter would complain about this, since typically I'd expect the object spread syntax instead.

```suggestion
    const props = {
      ...DEFAULT_PROPS,
      value: {
        personalKey: '',
        personalKeyConfirm: '1234-asdf',
      },
      toPreviousStep: () => {},
    };
```

Reply:
> I would have thought the linter would complain about this

And now I'm quite confused that it doesn't, because we definitely have [`prefer-object-spread`](https://eslint.org/docs/latest/rules/prefer-object-spread) configured to error 🤔 

```
$ yarn eslint --print-config .eslintrc | grep -A 2 prefer-object-spread
    "prefer-object-spread": [
      "error"
    ],
```